tools/helper.psm1

```diff
@@ -0,0 +1,292 @@
+
+using namespace System.Runtime.InteropServices
+
+$IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
+$MinimalSDKVersion = '2.1.300'
+$RepoRoot = (Resolve-Path "$PSScriptRoot/..").Path
+$LocalDotnetDirPath = if ($IsWindowsEnv) { "$env:LocalAppData\Microsoft\dotnet" } else { "$env:HOME/.dotnet" }
+
+function Find-Dotnet
+{
+    $dotnetFile = if ($IsWindowsEnv) { "dotnet.exe" } else { "dotnet" }
+    $dotnetExePath = Join-Path -Path $LocalDotnetDirPath -ChildPath $dotnetFile
+
+    # If dotnet is already in the PATH, check to see if that version of dotnet can find the required SDK.
+    # This is "typically" the globally installed dotnet.
+    $foundDotnetWithRightVersion = $false
+    $dotnetInPath = Get-Command 'dotnet' -ErrorAction SilentlyContinue
```

Comment:
You should prefer `Ignore` to avoid polluting $error.

Reply:
Good point. Updated.